lib/hooks/form-builder/useHandleAdd.tsx

```diff
@@ -34,6 +35,7 @@ export const useHandleAdd = () => {
 
   const create = useCallback(async (type: FormElementTypes) => {
     const defaults = JSON.parse(JSON.stringify(defaultField));
```

Comment:
Do we need JSON.stringify here?

Reply:
not sure, that was there previously. can test it since I'm in here.